src/Message/FetchSubscriptionInvoiceRequest.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+namespace Omnipay\Vindicia\Message;
+
+use stdClass;
+use Omnipay\Common\Exception\InvalidRequestException;
+
+/**
+ * Fetch a Vindicia subscription invoice object.
+ *
+ * Parameters:
+ * - subscriptionId: Your identifier for the subscription to be fetched. Either subscriptionId
+ * or subscriptionReference is required.
+ * - subscriptionReference: The gateway's identifier for the subscription to be fetched. Either
+ * subscriptionId or subscriptionReference is required.
+ * - invoiceReference: the invoice number which uniquely identifies the fetched invoice within subscription
```

Comment:
can you mention that this should be obtained from FetchSubscriptionInvoiceReferencesRequest?